app/assets/stylesheets/ama_layout/layout/helper-classes.scss

```diff
@@ -218,6 +218,23 @@
   background-color: rgba(255,255,255,0.6);
 }
 
+.vr{
+  &--desktop{
+    background-color: $stone;
+    width: 2px;
+    height: 100%;
+    position: absolute;
+    right: 0;
+    top: 0;
+
+    @include breakpoint(small down){
```

Comment:
we should make the mobile view a modifier instead. Then we can choose whether it's applied in other situations to help us future proof this a bit.